### DISCUSSION TITLE
Move the view switcher from navigation bar to the header

### DISCUSSION THREAD

**Discussion by math-GH** [FreshRSS/FreshRSS#5036] [Ideas]
Hi,

I have been struggeling with the view buttons in the navigation bar since my first day.
![grafik](https://user-images.githubusercontent.com/1645099/213940028-a9bb1468-32c1-4643-8baa-2e1de35655cf.png)

I understand the menu bar as options for the current view/list. And I understand the different views as different "pages", so they do not fit well into the menu bar.

I would like to move the 3 view buttons up to the header:
![grafik](https://user-images.githubusercontent.com/1645099/213940097-5772bff7-805a-4a67-b626-dabf46e54824.png)

Why I would like to see benefits:
- for me it would improve the UX
- it would improve the menu bar in the mobile view, that has now 2 rows on my phone

What do you think?

**Comment by Frenzie** [comment]
Seems fine to me. Why not the RSS feed button?

**Reply by math-GH** [reply]
The RSS button is not a view. It is the export of the filter results shown in the list below. It is the "export" of the view.

**Reply by Frenzie** [reply]
I was just curious, but now it seems you made a great argument for why they should definitely be together, heh.

**Reply by math-GH** [reply]
I don't get it...

**Reply by Frenzie** [reply]
Because "of the view".

**Comment by Alwaysin** [comment]
This change would make sense to me. It makes the menu bar less "impressive" and more user-friendly.
I would ever go further than this and put them in the settings page, or are there really people fiddling with the view modes that much?

**Reply by math-GH** [reply]
yes there are use cases that needs the switch from normal view to global view to reader view

**Comment by Alkarex** [comment]
I do not have any objection as long as it also works fine in mobile view, and does not break extensions using the `freshrss_nav_menu` hook.
